frontend/sentiment-dashboard/src/environments/environment.ts

```diff
@@ -4,6 +4,14 @@
 
 export const environment = {
   production: false,
+  amplify: {
```

Comment:
TODO for the future: figure out whether 1. these values should be secret and 2. how we should store them.